src/tasks/index.ts

```diff
@@ -28,16 +28,25 @@ export async function execute (job: Job) {
     scenario = new ProjectScenario()
   }
 
-  // Setup RUNBOX
-  await scenario.setup(currentJobDir, job) 
+  let result
+  try {
+    // Setup RUNBOX
+    await scenario.setup(currentJobDir, job)
 
-  // Run worker
-  await scenario.run(currentJobDir, job)
+    // Run worker
+    await scenario.run(currentJobDir, job)
 
-  // Get result
-  const result = await scenario.result(currentJobDir, job)
+    // Get result
+    result = await scenario.result(currentJobDir, job)
+  }
+  catch(err) {
+    result = {
+      id: job.id,
+      stderr: err
+    }
```

Comment:
What do you think of this `try` `catch` block. Earlier if there was any error in the code, the code would break and server would stop. So I think `try` `catch` is required

There is one problem with returning only `id` and `stderr`. The [judge-api-queue-logic](https://github.com/coding-blocks/judge-api/blob/30640fe287d09f6ce01088062e501e4817da9119/src/rabbitmq/jobqueue.ts#L65) will fail and it will always think that this is `run_result` event

Reply:
We do have an error queue mechanism so won't be needing try catch here